arch/Cargo.toml

```diff
@@ -13,6 +13,7 @@ tdx = []
 acpi_tables = { path = "../acpi_tables", optional = true }
 anyhow = "1.0"
 byteorder = "1.4.3"
+fdt_parser = { version = "0.1.3", package = 'fdt'}
```

Comment:
Is this the crate: https://crates.io/crates/fdt ?
I'm asking because this has been published very recently and there's only 519 downloads at the moment :-/

Reply:
/cc @rbradford wdyt?

Reply:
What about making this and the other fdt dependency architecture specific: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#platform-specific-dependencies

Reply:
@sboeuf Yes, that is it. Indeed it's a new crate.

@rbradford Sure, good idea.